### DISCUSSION TITLE
Feedback section

### DISCUSSION THREAD

**Discussion by PanJohnny** [modrinth/.github#4] [Feature requests]
### Is your feature request related to a problem? Please describe.

Other users can't give you feedback directly on modrinth.

### Describe the solution you'd like

Add feedback section to mod for github logged in users, so they can give you feedback. This would directly help the developer, when someone browsing reads the feedback.

### Describe alternatives you've considered

Add out link for developer to link feedback.

**Comment by Secret-chest** [comment]
> for github logged in users

If you have github, why not just post an issue? Maybe you could choose to enable it (disabled by default) if the mod is closed-source.

**Comment by ghost** [comment]
It's in the plans, i think? Could be simpler described as a comment section.
